services/graphql-server/src/graphql/resolvers/platform/content.js

```diff
@@ -482,7 +482,20 @@ module.exports = {
       if (section) return section;
 
       // Current section does not match site, load alternate.
-      // @todo This should eventually account for secondary sites/sections. For now, load home.
+      // @todo This should eventually account for secondary sites/sections.
+      // For now load an alternate from schedules.
+      const sectionQuery = getAsArray(content, 'sectionQuery');
+      if (sectionQuery.length) {
+        if (!siteId) throw new UserInputError('A site id must be set to generate the `Content.primarySection` field.');
+        const currentSiteSections = sectionQuery.map((schedule) => {
+          if (schedule.siteId === siteId) return schedule;
+          return null;
+        }).filter(v => v);
+        if (currentSiteSections.length) {
+          const currentSiteSection = await load('websiteSection', currentSiteSections[0].sectionId, projection, query);
+          if (currentSiteSection) return currentSiteSection;
+        }
+      }
```

Comment:
Same change here. Since you're doing the same thing in both files, export this logic as a utility function. It should take the `content` object and `load` function as parameters, and return the found section object (if present).

In the path resolver, you'd just return the alias from the secondary section, if it was found.